### PR TITLE
Feature authorship

### PR DESCRIPTION
**Pull request by jprestop**
## Pull Request Testing ##

- [x] Describe testing already performed for these changes:</br>
Only documentation changes, mostly about the authorship list.

- [x] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>
No testing, just review the doc changes.

- [x] Do these changes include sufficient documentation updates, ensuring that no errors or warnings exist in the build of the documentation? **[Yes]**

- [x] Do these changes include sufficient testing updates? **[Yes]**
None needed

- [x] Will this PR result in changes to the test suite? **[No]**</br>
If **yes**, describe the new output and/or changes to the existing output:</br>

- [x] Please complete this pull request review by **[May 7, 2021]**.</br>

## Pull Request Checklist ##
See the [METplus Workflow](https://dtcenter.github.io/METplus/Contributors_Guide/github_workflow.html) for details.
- [x] Complete the PR definition above.
- [x] Ensure the PR title matches the feature or bugfix branch name.
- [x] Define the PR metadata, as permissions allow.
Select: **Reviewer(s)**, **Project(s)**, and **Milestone**
- [ ] After submitting the PR, select **Linked Issues** with the original issue number. **No issue**
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue and delete your feature or bugfix branch from GitHub.

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -21,7 +21,7 @@
 project = 'METplotpy'
 copyright = '2021, NCAR'
 author = 'UCAR/NCAR, NOAA, CSU/CIRA, and CU/CIRES'
-author_list = 'Win-Gildenmeister, M., Burek, T., Fillmore, D., Halley Gotway, J., Adriaansen, D., Fisher, H.'
+author_list = 'Win-Gildenmeister, M., Burek, T., Fillmore, D., Adriaansen, D., Fisher, H.'
```
Comment (bikegeek) [inline docs/conf.py]: please add Christina Kalb to the list of authors

File: docs/index.rst [hunk 2]
```diff
@@ -90,7 +90,27 @@ Acronyms
...
+
+Many authors, listed below in alphabetical order, have contributed to the documentation of METplus.
+To cite this documentation in publications, please refer to the METplotpy User's Guide :ref:`Citation Instructions<citations>`.
+
+* Daniel Adriaansen [#NCAR]_
+* Tatiana Burek [#NCAR]_
+* David Fillmore [#NCAR]_
+* Hank Fisher [#NCAR]_
+* Minna Win-Gildenmeister [#NCAR]_
+
```
Comment (bikegeek) [inline docs/index.rst:106]: please add Christina Kalb

**Review by bikegeek — COMMENTED** [review]
Please add Christina Kalb.  Comments are in-line where additions need to be made.  Otherwise, all else looks OK

File: docs/index.rst (hunk 2 above)
Comment (jprestop) [reply docs/index.rst:106]: Done

File: docs/conf.py (hunk 1 above)
Comment (jprestop) [reply docs/conf.py]: Done

**Comment by jprestop** [comment]
Thanks, Minna.  I made the requested corrections.

**Review by bikegeek — APPROVED** [review]
Thanks for making the updates.  Approved.

**Pull request merged**